### PR TITLE
Listing Orbs now optionally accepts the '--private' flag

### PR DESCRIPTION
**Pull request by kelvinkfli**
This PR allows CLI users to return private orbs within a given namespace by providing the `--private` flag. When this flag is set, the user must also provide a namespace, otherwise an error will be returned.

Co-authored-by: @ryanbarkley <ryanbarkley@users.noreply.github.com>

### CODE REVIEW

**Review by skimke — APPROVED** [review]

File: cmd/orb_test.go [hunk 1]
```diff
@@ -2158,7 +2161,104 @@ query namespaceOrbs ($namespace: String, $after: String!) {
...
+				Expect(string(stdout)).To(Equal(`Orbs found: 5. Showing only certified orbs.
+Add --uncertified for a list of all orbs.
+
+circleci/delete-me (Not published)
+circleci/delete-me-too (Not published)
+circleci/gradle (0.0.1)
+circleci/heroku (Not published)
+circleci/rollbar (0.0.1)
+
+In order to see more details about each orb, type: ` + "`circleci orb info orb-namespace/orb-name`" + `
```
Comment (skimke) [inline cmd/orb_test.go:2256]: I'm a little worried that they may run this on private orbs :/ but I'm also not sure that we want to introduce temporary messaging.

File: cmd/orb_test.go [hunk 2]
```diff
@@ -2158,7 +2161,104 @@ query namespaceOrbs ($namespace: String, $after: String!) {
...
+
+Search, filter, and view sources for all Orbs online at https://circleci.com/developer/orbs/
```
Comment (skimke) [inline cmd/orb_test.go:2258]: This part would benefit from saying "except private orbs"

**Review by skimke — APPROVED** [review]
LGTM - a few questions for you on the messages returned when successfully listing orbs

File: cmd/orb_test.go (hunk 1 above)
Comment (kelvinkfli) [reply cmd/orb_test.go]: I agree. I re-wrote the existing function responsible for constructing this response, let me know what you think!

File: cmd/orb_test.go (hunk 2 above)
Comment (kelvinkfli) [reply cmd/orb_test.go]: In the new formatting function, I opted to just not prompt users (until we've reached parity) to take any action and eliminate these lines when `--private` is provided. Any thoughts?

Comment (skimke) [reply cmd/orb_test.go]: Make sense, thanks for updating!

**Pull request merged**